cfy_manager/constants.py

```diff
@@ -31,6 +31,8 @@
 SUDOERS_INCLUDE_DIR = '/etc/sudoers.d'
 CLOUDIFY_SUDOERS_FILE = join(SUDOERS_INCLUDE_DIR, CLOUDIFY_USER)
 INITIAL_INSTALL_DIR = join(CLOUDIFY_HOME_DIR, '.installed')
+INSTALLED_COMPONENTS = join(INITIAL_INSTALL_DIR, 'components.yaml')
+INSTALLED_PACKAGES = join(INITIAL_INSTALL_DIR, 'packages.yaml')
```

Comment:
It looks like we only keep these yaml files so we can update them, but don't actually consume the data in them, so I'm still leaning towards it being better for us to just delete them- that'd also save a bit of code.

Is there any really compelling reason to keep them, given that if we do then someone might start using them and make it harder for us to remove them later?

Reply:
I think my answer here: https://github.com/cloudify-cosmo/cloudify-manager-install/pull/940#discussion_r497397165 is also relevant for this question.

Reply:
OK, will mark this as resolved and continue the conversation there.